### PR TITLE
move logo folder into docs

### PR DESCRIPTION
**Pull request by Debiday**
Moved the logo folder into "docs" to help the main folder look cleaner.

### CODE REVIEW

**Comment by Debiday** [comment]
Hi @stackedsax, can I trouble you to have a look? Thank you! 😊

**Review by jgiannuzzi — COMMENTED** [review]
Hey @Debiday, sorry for the late review!

May I please ask you to also remove the logo that sits at the root and update the README to use the same logo, but from the new `docs/logo` folder?

Otherwise, LGTM 👍

**Comment by Debiday** [comment]
Thank you @jgiannuzzi for the review! I have updated the logo now –you will notice two commits, [b6f5fcc](https://github.com/G-Research/thanos-remote-read/pull/24/commits/b6f5fcc7397d72586418390409cf54397b77b98f) to test the logo appearance from the forked branch, and [f256dc9](https://github.com/G-Research/thanos-remote-read/pull/24/commits/f256dc9244e13310defbb1e33612e0f17ec49fea), to update the src file with the correct location after the files get merged. You will see an error in this fork 
Screen Shot 2022-02-19 at 5 25 50 PM
but once merged it should appear correctly, if not I can always go back to update the src location of the file. Does this make sense?

**Comment by Debiday** [comment]
@jgiannuzzi and also deleted the file in the root folder now, thank you!

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<img src="Thanos-remote-read_SignatureLogo_RGB-Black.svg" width="180" alt="thanos-remote-read">
+<img src="https://github.com/G-Research/thanos-remote-read/blob/logo-folder/docs/logo/png/Thanos-remote-read_SignatureLogo_RGB-Black.png" width="180" alt="thanos-remote-read">
```
Comment (jgiannuzzi) [inline README.md:1]: I think we want the source to be relative instead, and we want to use the vector version. The suggestion below _should_ work on both your repo and this one.

```suggestion
<img src="docs/logo/svg/Thanos-remote-read_SignatureLogo_RGB-Black.svg" width="180" alt="thanos-remote-read">
```

**Review by jgiannuzzi — CHANGES_REQUESTED** [review]
Thanks for the changes @Debiday!

Have a look at the suggestion below to make it work in all cases, as currently it wouldn't work, even after a merge.

File: README.md (hunk 1 above)
Comment (Debiday) [reply README.md]: Doy! Yes, you are right. I've committed your suggestion now, thanks😊

**Comment by Debiday** [comment]
Doy! Yes, you are right @jgiannuzzi. I've committed your suggestion now, thanks for the catch 😊

**Review by jgiannuzzi — APPROVED** [review]

**Comment by jgiannuzzi** [comment]
Thanks for your PR, @Debiday 👍

**Pull request merged**